CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Switched `--no-animation` as a public option.
```

Comment:
Please write the changelog correctly.
Whrere is the category? Is it "Switched"?

Reply:
OK